clients/web/src/views/body/CollectionsView.js

```diff
@@ -94,6 +96,25 @@ var CollectionsView = View.extend({
         collection.set('_id', result.id).on('g:fetched', function () {
             router.navigate('/collection/' + collection.get('_id'), {trigger: true});
         }, this).fetch();
+    },
+
+    _toggleDescription: function (e) {
+        const link = $(e.currentTarget);
+        const cid = link.attr('cid');
+        const dest = this.$(`.g-collection-description[cid="${cid}"]`);
```

Comment:
If you are interested in seeing all the greatness of modern ES, [this](http://es6-features.org/) is a really nice resource.